### ISSUE TITLE
django-material-admin having trouble with Django==3.2 release

### ISSUE THREAD

**Issue by Vulum** [MaistrenkoAnton/django-material-admin#137]
It seems that django-material-admin is having trouble with Django==3.2 release

Error that I am receiving is as following:

```
/var/www/html/gesi/backend/src/gesi/settings.py changed, reloading.
Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/var/www/html/gesi/backend/src/env/lib/python3.6/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/var/www/html/gesi/backend/src/env/lib/python3.6/site-packages/django/core/management/commands/runserver.py", line 110, in inner_run
    autoreload.raise_last_exception()
  File "/var/www/html/gesi/backend/src/env/lib/python3.6/site-packages/django/utils/autoreload.py", line 87, in raise_last_exception
    raise _exception[1]
  File "/var/www/html/gesi/backend/src/env/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    autoreload.check_errors(django.setup)()
  File "/var/www/html/gesi/backend/src/env/lib/python3.6/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/var/www/html/gesi/backend/src/env/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/var/www/html/gesi/backend/src/env/lib/python3.6/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/var/www/html/gesi/backend/src/env/lib/python3.6/site-packages/django/apps/config.py", line 255, in create
    return app_config_class(app_name, app_module)
  File "/var/www/html/gesi/backend/src/env/lib/python3.6/site-packages/django/apps/config.py", line 39, in __init__
    "The app label '%s' is not a valid Python identifier." % self.label
django.core.exceptions.ImproperlyConfigured: The app label 'material.admin' is not a valid Python identifier.
```

Any reason why that is happening? Thank you in advance........

**Comment by oberix** [comment]
It's seems to be because of https://docs.djangoproject.com/en/3.2/releases/3.2/#automatic-appconfig-discovery .

The AppConfig.label shoudn't be `materia.admin` but some string which is a valid python identifier. When I tried to change it to admin, or simply comment it, another exception occured:

```
django.core.exceptions.ImproperlyConfigured: Application labels aren't unique, duplicates: admin
```

Which should to be `material.admin.default`

**Comment by Vulum** [comment]
Hi @oberix, thanks for responding, you implying removing 'material.admin' from the settings?

**Comment by oberix** [comment]
Hi @Vulum, that of course would make your project run... but that's just "killing the patient to cure the disease"!

No, the issue here is in the `AppConfig` class of django-material-admin. I did just a few quick attempt to fix it but could only stumble in another (related?) problem. Sorry but I don't have the time and the urgency now to go deeper into this. Just thought that it would be a good pointer to someone more into the development of this app.

**Comment by Vulum** [comment]
Appreciate your response @oberix

**Comment by MaistrenkoAnton** [comment]
in progress, will be ready this week
https://github.com/MaistrenkoAnton/django-material-admin/pull/138

**Comment by Vulum** [comment]
Hi @MaistrenkoAnton , truly appreciate your work and effort

**Comment by Voxmundi** [comment]
I'm here for the same bug. It's a great work. Waiting forward to be fixed.

**Comment by MaistrenkoAnton** [comment]
fixed since `v1.8.3`
